shell/platform/android/android_shell_holder.cc

```diff
@@ -271,6 +271,24 @@ void AndroidShellHolder::NotifyLowMemoryWarning() {
   shell_->NotifyLowMemoryWarning();
 }
 
+void AndroidShellHolder::DrawLastLayerTree() {
+  FML_DCHECK(shell_);
+  thread_host_->raster_thread->GetTaskRunner()->PostTask([&]() {
```

Comment:
This is potentially unsafe if the queued task could run after the `AndroidShellHolder` is deleted.